.github/styles/Nuclio/ignore.txt

```diff
@@ -0,0 +1,73 @@
+dev
+kubeconfig
+MLRun
+Kubeflow
+serverless
+Katacode
```

Comment:
Don't thing that's a real thing (`Katacoda` is, and it is already in the list below)